Alerting plugin - experimental cross cluster monitor support documentation (#6350)
opensearch-project/documentation-website: _observing-your-data/alerting/per-cluster-metrics-monitors.md

The cluster metrics monitor supports up to **ten** triggers.

> In the following example, the monitor is configured to call the cluster health API for two clusters, (`cluster-1`, and `cluster-2`). The trigger condition will create an alert when either of the clusters' `status` is not `green`. 

Reviewer: ```suggestion
In the following example, the monitor is configured to call the Cluster Health API for two clusters, (`cluster-1` and `cluster-2`). The trigger condition will create an alert when either of the clusters' `status` is not `green`. 
```

> `script` points the `source` to the Painless script `for (cluster in ctx.results[0].keySet()) if (ctx.results[0][cluster].status != \"green\") return true`. See [Trigger variables]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/#trigger-variables) for more `painless ctx` variable options.

Reviewer: ```suggestion
The `script` parameter points the `source` to the Painless script `for (cluster in ctx.results[0].keySet()) if (ctx.results[0][cluster].status != \"green\") return true`. See [Trigger variables]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/#trigger-variables) for more `painless ctx` variable options.
```

  ]
}
```

> The dashboards interface supports selecting clusters to monitor, and the desired API.

Reviewer 2: ```suggestion
The dashboards interface supports the selection of clusters to be monitored and the desired API.
```
